pwa-devdocs/src/_data/buildpack-api.yml

```diff
@@ -1,5 +1,17 @@
 title: PWA Buildpack API
 entries:
+    - label: BuildBus
+      url: /pwa-buildpack/reference/buildbus/
+      entries:
+        - label: Extensibility Targets
+          url: /pwa-buildpack/reference/buildbus/targets/
```

Comment:
I know that I called the directory `BuildBus` in source code, but I think in documentation, the top-level item should be called "Extensibility" and there should be an additional entry below, called "BuildBus".